extension/view/js/InPageTranslation.js

```diff
@@ -290,7 +290,9 @@ class InPageTranslation {
     }
 
     isElementHidden(element) {
-        return element.style.display === "none" || element.style.visibility === "hidden" || element.offsetParent === null;
+        return window.getComputedStyle(element).display === "none" ||
```

Comment:
Let's call `getComputedStyle` once instead of twice.

```js
let style = window.getComputedStyle(element);
return style.display === "none" || style.visibility = etc.
```